ci_scripts/generate_ios26_testplan.rb

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env ruby
+
+require 'json'
+require 'find'
+
+class IOS26TestPlanGeneratorV2
+  def initialize
+    @test_plan_file = File.join(File.dirname(__dir__), 'Stripe', 'AllStripeFrameworks-iOS26.xctestplan')
+    @tests_by_target = {}
+  end
+
+  def generate
+    log_info "Starting iOS 26 test plan generation (v2)..."
+    
+    # Find all files containing @iOS26
+    annotated_files = find_annotated_files
+    
+    if annotated_files.empty?
+      log_warning "No files with @iOS26 annotations found"
+      return
+    end
+    
+    log_info "Found #{annotated_files.length} files with @iOS26 annotations"
+    
+    annotated_files.each { |file| process_annotated_file(file) }
+    
+    write_test_plan
+    log_results
+  end
+
+  private
+
+  def find_annotated_files
+    files = []
+    Find.find('.') do |path|
+      next unless path.end_with?('.swift')
+      next unless path.include?('/Test') || path.include?('Test.swift') || path.include?('Tests.swift')
+      
+      # Check if file contains @iOS26 annotation
+      if File.read(path, encoding: 'UTF-8').include?('// @iOS26')
+        files << path
+      end
+    rescue => e
+      log_warning "Could not read file #{path}: #{e.message}"
+    end
+    
+    files.sort
+  end
+
+  def process_annotated_file(file_path)
+    log_info "Processing: #{file_path}"
+    
+    content = File.read(file_path, encoding: 'UTF-8')
+    lines = content.lines
+    
+    # Determine target name from file path
+    target_name = determine_test_target(file_path)
+    unless target_name
+      log_info "  ↳ Could not determine test target for #{file_path}"
+      return
```

Comment:
Should we error out if we can't find the test target... and maybe log_error might be more appropriate?

Reply:
Yeah, good call. I'll make most of these failures error and log_error.